.circleci/config.yml

```diff
@@ -26,7 +26,6 @@ jobs:
       tag: 14-stretch-browsers
     steps:
       - checkout
-      - node/install-npm
```

Comment:
What motivated you to make this change?

Reply:
The build was failing on CI due to issues installing Cypress. I thought this was due to having multiple versions of npm or Cypress installed but I'm not sure if this is actually the correct fix - was hoping someone might be able to give me a hand with this today.

Reply:
Have removed this to spend some more time on it today.